### PR TITLE
feat: adding jwmdev bootstrap node to config

### PR DESCRIPTION
**Pull request by jwmdev**
## Description

Added boostrap node in the config

## Related issue(s)

If this Pull Request is related to an issue, mention it here.
- Fixes #(issue number)

### CODE REVIEW

File: config/config.go [hunk 1]
```diff
@@ -115,6 +115,7 @@ func DefaultConfigTestnet() *Config {
 		"/ip4/209.250.235.91/tcp/21777/p2p/12D3KooWETQgcTCFv2kejUsGMVVmnkNoTW8wh33MevAzyeYYzQkr",      // Mr HoDL (1llusiv387@gmail.com)
 		"/dns/pactus.nodesync.top/tcp/21777/p2p/12D3KooWP25ejVsd7cL5DvWAPwEu4JTUwnPniHBf4w93tgSezVt8", // NodeSync.Top (lthuan2011@gmail.com)
 		"/ip4/95.217.89.202/tcp/21777/p2p/12D3KooWMsi5oYkbbpyyXctmPXzF8UZu2pCvKPRZGyvymhN9BzTD",       // CodeBlockLabs (emailbuatcariduit@gmail.com)
+		"/ip4/65.109.11.208/tcp/21777/p2p/12D3KooWQ2cfLX1kcXhAzcb23sHWHaK5DBkk2xtYTZu8JbwG97K4",
```
Comment (kehiy) [inline config/config.go:118]: @jwmdev looks good to me.
I think you can add your name and email too.

**Review by kehiy — APPROVED** [review]

**Comment by b00f** [comment]
@jwmdev @kehiy 

Why there is no name and email contact there?
@jwmdev Please add it in the next PR. Thanks

**Comment by kehiy** [comment]
@b00f ok. I thought it's optional and just suggest it to joseph.

**Comment by b00f** [comment]
No. Please make sure @jwmdev adds the email and name.

**Comment by kehiy** [comment]
Ok, I'm waiting for him to respond.

**Pull request merged**